third_party/amd/lib/TritonAMDGPUToLLVM/TritonGPUToLLVM.cpp

```diff
@@ -173,6 +173,8 @@ struct ConvertTritonAMDGPUToLLVM
                                                axisInfoAnalysis, AMDBenefit);
     mlir::triton::populateConvertLayoutOpToLLVMPatterns(
         typeConverter, targetInfo, patterns, commonBenefit);
+    mlir::triton::populateConvertLayoutOpUsingLinearLayoutsToLLVMPattern(
+        typeConverter, targetInfo, patterns, commonBenefit + 1);
```

Comment:
why adding it here? Should this already be done by the line above

Reply:
you mean it should be in AMD::populateConvertLayout.., it cannot be in mlir::triton::populateConvert...

Reply:
oh, it can actually, nevermind

Reply:
I'm confused why anything needs to change?

Reply:
Chatted offline, seems now its correct.